Timing-Functions/ParseDate.js

```diff
@@ -0,0 +1,27 @@
+import { getMonthDays } from './GetMonthDays'
+
+function checkDate(date) {
```

Comment:
I would not make this a separate function, just inline it.

Reply:
Because there are many ways, in which the `date` can be incorrect, I am more in a favor of leaving it as a separate function.